exporter/signalfxexporter/translation/converter.go

```diff
@@ -411,12 +386,12 @@ func float64ToDimValue(f float64) string {
 // resourceAttributesToDimensions will return a set of dimension from the
 // resource attributes, including a cloud host id (AWSUniqueId, gcp_id, etc.)
 // if it can be constructed from the provided metadata.
-func appendResourceAttributesToDimensions(dims []*sfxpb.Dimension, resourceAttr map[string]string) []*sfxpb.Dimension {
+func appendResourceAttributesToDimensions(dims []*sfxpb.Dimension, resourceAttr pdata.AttributeMap) []*sfxpb.Dimension {
```

Comment:
Based on the caller (visible in the diff) I think you can make this just return the array and remove the argument dims.

Reply:
Done.